pandas/tests/window/test_rolling.py

```diff
@@ -919,6 +919,67 @@ def test_rolling_var_numerical_issues(func, third_value, values):
     tm.assert_series_equal(result, expected)
 
 
+def test_timeoffset_as_window_parameter():
+
```

Comment:
Could you leave a comment with the github issue number? `# GH 28266`

Reply:
Done!